101-DDoS-Attack-Prevention/DSC/Configure-AzureSecurityCenter.ps1

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
these aren't DSC scripts, why are they in the DSC folder?

Reply:
this will be used to configure the security vault the the user subscription.